editor.js

```diff
@@ -0,0 +1,4 @@
+ 
```

Comment:
I could not see what HTML file includes this .js file.

Reply:
Just changing the name sir.
By mistake I added index.js in place of editor.js